pom.xml

```diff
@@ -196,34 +196,35 @@
     </repository>
   </repositories>
 
-  <profiles>
+   <profiles>
     <profile>
-      <id>release</id>
-      <properties>
-        <plugin.version.gpg>1.6</plugin.version.gpg>
-        <gpg.passphrase>${env.GPG_PASS}</gpg.passphrase>
-        <resume>false</resume>
-        <tag>${env.RELEASE_VERSION}</tag>
-        <releaseVersion>${env.RELEASE_VERSION}</releaseVersion>
-        <developmentVersion>${env.DEVELOPMENT_VERSION}</developmentVersion>
-        <arguments>-DskipTests</arguments>
-      </properties>
+      <id>community-action-maven-release</id>
```

Comment:
:x: FYI this name is referenced in the GitHub action https://github.com/camunda-community-hub/zeebe-process-generator/blob/3b758dbeb2f81d39e24629b4aa558dcca1418395/.github/workflows/ci.yml#L54

I will add a commit for this.

Reply:
Commit added